api/src/controllers/auth.ts

```diff
@@ -232,6 +233,21 @@ router.get(
 			req.session.redirect = req.query.redirect as string;
 		}
 
+		const accountability = {
+			ip: req.ip,
+			userAgent: req.get('user-agent'),
+			role: null,
+		};
```

Comment:
We should be able to rely on `req.accountability` instead of having to create one ourselves